spanner/spec/spanner_postgresql_jsonb_add_column.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "./spec_helper"
+require_relative "./spanner_postgresql_helper"
+require_relative "../spanner_postgresql_jsonb_add_column"
+
+describe "Google Cloud Spanner Postgres examples" do
+  before :each do
+    cleanup_database_resources
+  end
+
+  after :each do
+    cleanup_database_resources
+    cleanup_instance_resources
```

Comment:
Just curious, where do we create instance resources?

Reply:
Instance is not created in the spec.. it is passed on via env variable `GOOGLE_CLOUD_SPANNER_TEST_INSTANCE `

Reply:
Just realised we don't have to cleanup instance resource here as we don't create any for this particular test. 
Thanks for pointing out dint realise it earlier 👍